src/panels/config/integrations/integration-panels/bluetooth/dialog-bluetooth-device-info.ts

```diff
@@ -81,10 +81,8 @@ class DialogBluetoothDeviceInfo extends LitElement implements HassDialog {
             ${Object.entries(this._params.entry.manufacturer_data).map(
               ([key, value]) => html`
                 <tr>
-                  ${key}
-                </tr>
-                <tr>
-                  ${this.showDataAsHex(value)}
+                  <td><b>${key}</b></td>
+                  <td>${this.showDataAsHex(value)}</td>
```

Comment:
Not sure I love `<td>` here. We need a good way to differentiate the manufacturer id (key) from the data